server/src/app.js

```diff
@@ -3,16 +3,20 @@ const createError = require("http-errors");
 const express = require("express");
 const path = require("path");
 const cookieParser = require("cookie-parser");
-const logger = require("morgan");
+const morgan = require("morgan");
 const helmet = require("helmet");
 const cors = require("cors");
+const logger = require("./logger");
 
 const app = express();
 app.set("query parser", "simple");
 
-process.env.NODE_ENV !== "test" && app.use(logger("dev"));
+if (process.env.NODE_ENV !== "test") {
```

Comment:
Should the `else` on this use `app.use(morgan("dev"))`

Reply:
I'm not sure I understand; are you saying we should add an `else` condition here?

Reply:
Yeah, I'm not clear on what the behavior would be if the environment is test, but I would assume that we would still want to pipe the logs out to console in that circumstance.

Reply:
Isn't the original logic saying here that we don't want to log to the console when env is test? I also did investigate more; the `npm test` command sets the test env var (test fail, btw).


**With proposed `else` condition.**
```
> node-server@0.0.0 test
> NODE_ENV=test mocha 'src/**/*.test.js' --inline-diffs --exit



  index routes
    GET /v1/latest
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
GET /v1/latest 200 47.448 ms - 857159
      ✓ the response should be an object (70ms)
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
GET /v1/latest 200 27.224 ms - 857159
      ✓ the response should have a meta property
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
      ✓ the response should have a foiaList property
GET /v1/latest 200 18.480 ms - 857159
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
      1) the response should have a foiaList property with 424 requests
GET /v1/latest 200 15.059 ms - 857159


  3 passing (196ms)
  1 failing

  1) index routes
       GET /v1/latest
         the response should have a foiaList property with 424 requests:

      AssertionError: expected [ Array(421) ] to have a length of 424 but got
421
      actual expected

      421424

      at Context.<anonymous> (src/routes/v1/index/routes.test.js:33:41)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Without proposed `else` condition.**
```
> node-server@0.0.0 test
> NODE_ENV=test mocha 'src/**/*.test.js' --inline-diffs --exit



  index routes
    GET /v1/latest
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
      ✓ the response should be an object (64ms)
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
      ✓ the response should have a meta property
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
      ✓ the response should have a foiaList property
/Users/chris/Sites/50-a-foil-status/server/src/datastore/data
      1) the response should have a foiaList property with 424 requests


  3 passing (187ms)
  1 failing

  1) index routes
       GET /v1/latest
         the response should have a foiaList property with 424 requests:

      AssertionError: expected [ Array(421) ] to have a length of 424 but got
421
      actual expected

      421424

      at Context.<anonymous> (src/routes/v1/index/routes.test.js:33:41)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

And because I had trouble telling the difference, here's the diff:
```diff
9,10c9
< GET /v1/latest 200 47.448 ms - 857159
<       ✓ the response should be an object (70ms)
---
>       ✓ the response should be an object (64ms)
12d10
< GET /v1/latest 200 27.224 ms - 857159
16d13
< GET /v1/latest 200 18.480 ms - 857159
19d15
< GET /v1/latest 200 15.059 ms - 857159
22c18
<   3 passing (196ms)
---
>   3 passing (187ms)
```